SharpPcap/CaptureEventArgs.cs

```diff
@@ -26,18 +26,29 @@ namespace SharpPcap
     /// <summary>
     /// Capture event arguments
     /// </summary>
-    public class CaptureEventArgs : EventArgs
+    public ref struct CaptureEventArgs
```

Comment:
You may want to change this struct name, since it's not actually an `EventArgs` anymore

Reply:
thoughts on naming?

Reply:
`PacketCapture` maybe?

Reply:
Did you make this change?